appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: build{build}
+
+
+environment:
+  GherkinVersion: "3.3.0"
+  GherkinPackageVersion: "$(GherkinVersion)-ci$(APPVEYOR_BUILD_NUMBER)"
+
+
+nuget:
+  project_feed: true
+  disable_publish_on_pr: true
+
+configuration: Release
+
+
+before_build:
+  - appveyor DownloadFile https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
+  - appveyor DownloadFile https://raw.githubusercontent.com/appveyor/ci/master/scripts/nuget-restore.cmd
+  - nuget-restore dotnet\Gherkin.CSharp.sln
```

Comment:
The appveyor.yml file has to be in the root folder. So moving into the dotnet directory, you have to create the build for the gherkin-dotnet project.
It really depends how often you are pushing the changes to the other repos. If you are doing it only short before a release, it is then not a great help to update SpecFlow always to the latest version.